package.json

```diff
@@ -180,19 +180,22 @@
     "user-home": "2.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "8.3.5",
-    "@commitlint/config-conventional": "8.3.4",
-    "@commitlint/travis-cli": "8.3.5",
+    "@commitlint/cli": "12.0.1",
+    "@commitlint/config-conventional": "12.0.1",
+    "@commitlint/travis-cli": "12.0.1",
     "@semantic-release/apm-config": "8.0.0",
     "eslint-config-airbnb-base": "13.2.0",
     "eslint-plugin-import": "2.20.1",
-    "husky": "4.2.3",
+    "husky": "4.3.8",
     "jasmine-fix": "1.3.1",
     "rimraf": "3.0.2",
-    "semantic-release": "17.0.4"
+    "semantic-release": "17.4.2"
```

Comment:
Sorry, for being late, but I wanted to remove these because our CI already supports linting the commits and semantic-release